src/test/compile-fail/impl-wrong-item-for-trait.rs

```diff
@@ -12,7 +12,9 @@
 
 trait Foo {
     fn bar(&self);
-    const MY_CONST: u32;
+    //~^ NOTE original trait requirement
+    //~| NOTE original trait requirement
```

Comment:
That's normal.  Since there are actually three tests in this file, and two of them refer to that definition, you'll need one for each.